app/services/converters/json_schema.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Converters
+  # Converts a Json Schema specification to the JSON-LD format
+  class JsonSchema
+    ##
+    # Converts a file to the JSON-LD format.
+    #
+    # @param file [#path]
+    # @return [Hash]
+    def self.convert(file)
+      reader = RDF::JSON::Reader.open(file.path)
```

Comment:
This won't work, because `RDF::JSON::Reader` parses RDF/JSON data, not JSON Schema. JSON Schema is covered by the `JsonSchemaZip` converter. If individual JSON Schema uploads are needed, we better extract that functionality from `JsonSchemaZip`.

Also, this particular thing seems to be out of the scope of the node type inference feature.

Reply:
@excelsior Ok for the first part.

For the second and last comment, I would like to keep it here. I observed that the JSON file used in Issue #79 is not in the format the application needs. But after this processing, it changes to be the exact expected format.

Reply:
Ah, that file is in the RDF/JSON format indeed. Just rename the converter to `RdfJson` to avoid confusion.